### PR TITLE
Rollout improved example converter

### PR DESCRIPTION
**Pull request by t0yv0**
Rolls out improved TF example converter.

### CODE REVIEW

File: sdk/dotnet/License.cs [hunk 1]
```diff
@@ -11,31 +11,6 @@ namespace Pulumi.VSphere
 {
     /// <summary>
     /// Provides a VMware vSphere license resource. This can be used to add and remove license keys.
-    /// 
-    /// ## Example Usage
-    /// 
-    /// &lt;!--Start PulumiCodeChooser --&gt;
```
Comment (iwahbe) [inline sdk/dotnet/License.cs:17]: I'm not sure why we are loosing this one.

**Comment by t0yv0** [comment]
```
t0yv0@Antons-MacBook-Pro> jq '.|select(.ConversionResults.typescript.FailureSeverity==2)|.ConversionResults.typescript.FailureInfo' cov/byExample.json  | sort | uniq -c                                                                                        ~/bugs/ci-mgmt-889/pulumi-vsphere
   1 ": Cannot assign value ({ vpxClientLicenseLabel: string, workflow: string }) to attribute of type \"Optional<Map<string>>\" for resource \"vsphere::License\"; cannot traverse value of type string; cannot traverse value of type union(none, output(string), string, annotated(0xc001a7c2c0))"
   1 "Duplicate data \"vsphere_content_library_item\" configuration: A vsphere_content_library_item data resource named \"item\" was already declared at /e3.tf:5,1-43. Resource names must be unique per type in each module."
   1 "Function not yet implemented: Function format not yet implemented"
   1 "Function not yet implemented: Function tonumber not yet implemented; Function not yet implemented: Function values not yet implemented"
```

**Comment by t0yv0** [comment]
```
  "r": {
    "csharp": {
      "FailureSeverity": 2,
      "FailureInfo": ": Cannot assign value ({ vpxClientLicenseLabel: string, workflow: string }) to attribute of type \"Optional<Map<string>>\" for resource \"vsphere::License\"; cannot traverse value of type string; cannot traverse value of type union(none, output(string), string, annotated(0xc001a97bc0))",
      "Program": "",
      "TranslationCount": 1
    },
    "go": {
      "FailureSeverity": 2,
      "FailureInfo": ": Cannot assign value ({ vpxClientLicenseLabel: string, workflow: string }) to attribute of type \"Optional<Map<string>>\" for resource \"vsphere::License\"; cannot traverse value of type string; cannot traverse value of type union(none, output(string), string, annotated(0xc001ab1780))",
      "Program": "",
      "TranslationCount": 1
    },
    "java": {
      "FailureSeverity": 0,
      "FailureInfo": "",
      "Program": "package generated_program;\n\nimport com.pulumi.Context;\nimport com.pulumi.Pulumi;\nimport com.pulumi.core.Output;\nimport com.pulumi.vsphere.License;\nimport com.pulumi.vsphere.LicenseArgs;\nimport java.util.List;\nimport java.util.ArrayList;\nimport java.util.Map;\nimport java.io.File;\nimport java.nio.file.Files;\nimport java.nio.file.Paths;\n\npublic class App {\n    public static void main(String[] args) {\n        Pulumi.run(App::stack);\n    }\n\n    public static void stack(Context ctx) {\n        var licenseKey = new License(\"licenseKey\", LicenseArgs.builder()        \n            .licenseKey(\"452CQ-2EK54-K8742-00000-00000\")\n            .labels(Map.ofEntries(\n                Map.entry(\"vpxClientLicenseLabel\", \"Hello World\"),\n                Map.entry(\"workflow\", \"Hello World\")\n            ))\n            .build());\n\n    }\n}\n",
      "TranslationCount": 1
    },
    "python": {
      "FailureSeverity": 2,
      "FailureInfo": ": Cannot assign value ({ vpxClientLicenseLabel: string, workflow: string }) to attribute of type \"Optional<Map<string>>\" for resource \"vsphere::License\"; cannot traverse value of type string; cannot traverse value of type union(none, output(string), string, annotated(0xc001a7dec0))",
      "Program": "",
      "TranslationCount": 1
    },
    "typescript": {
      "FailureSeverity": 2,
      "FailureInfo": ": Cannot assign value ({ vpxClientLicenseLabel: string, workflow: string }) to attribute of type \"Optional<Map<string>>\" for resource \"vsphere::License\"; cannot traverse value of type string; cannot traverse value of type union(none, output(string), string, annotated(0xc001a7c2c0))",
      "Program": "",
      "TranslationCount": 1
    },
    "yaml": {
      "FailureSeverity": 0,
      "FailureInfo": "",
      "Program": "resources:\n  licenseKey:\n    type: vsphere:License\n    properties:\n      licenseKey: 452CQ-2EK54-K8742-00000-00000\n      labels:\n        - vpxClientLicenseLabel: Hello World\n          workflow: Hello World\n",
      "TranslationCount": 1
    }
  }
}

```

**Comment by t0yv0** [comment]
Specifically for license.

**Pull request merged**